examples/tutorials/Introduction_to_ProtBERT.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<p>It is possible to explain a bit more the meaning of this result ?  Is there  a comparison  with other method  ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>The result showcases how well the current model predicts soluble vs non-soluble protein correctly. There are other works [1] [2]  that we can compare against but in this tutorial we are not training on the complete DeepLoc data since thats computationally its very expensive, so comparing results against other works might be unfair. However, to check if the implementation was correct I did run this notebook on a dedicated GPU server we were able to match the results published in ProtTrans with a margin of 5%, but it takes a few hours for the training to complete.</p><p> </p><p>[1] Heinzinger, Michael, et al. "Modeling aspects of the language of life through transfer-learning protein sequences." <em>BMC bioinformatics</em> 20 (2019): 1-17.</p><p>[2] Thumuluri, Vineet, et al. "DeepLoc 2.0: multi-label subcellular localization prediction using protein language models." <em>Nucleic acids research</em> 50.W1 (2022): W228-W234.</p>

Reply:
<p>I have added some explanation regarding the final result.</p>